_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
+layout: post
+date: 2015-02-18 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+We have recently received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation.
+
+We care about your feedback, and we think that documentation and tutorials about [API Blueprint](http://apiblueprint.org) are extremely important.
+
+One of the most requested features was to have the ability to look to the other blueprints as a source of inspiration, and today this is possible for all public API Blueprints.
+
+Whenever you're surfing a public blueprint, a new **API Blueprint** button will appear in header bar. Clicking this button will open up a new window with readable markdown, that can be easily grabbed, studied, modified and republished.
+
+<img src="/images/2015-02-18-ApiBlueprint-button/apiblueprintbutton.png" />
+
+Sharing your API Blueprint with the community is very easy. Simply go to your API settings and ensure "Public Blueprint" switch is turned on.
+
+<img src="/images/2015-02-18-ApiBlueprint-button/settings.png" />
+
+We have many other great features on the horizon as well. As next step, we plan to merge this experience with our editor so that in a single click, you can open and edit the shared blueprint using our powerful tools.
```

Comment:
Suggestion to replace the first sentence: 'Socializing and learning from other public blueprints won't stop here. We have so many other great features in store to help promote learning through collaboration. Next, we plan.... "